### PR TITLE
feat(ValidCreatableSelect): separate CreatableSelect from Select

### PR DESCRIPTION
**Pull request by aris-fm**
This is based on feedback from [this PR](https://github.com/AccelByte/justice-ui-library/pull/64)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,109 +4,163 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.6.1]
```
Comment (rayhan-ab) [inline CHANGELOG.md:7]: i think you should revert the changelog, turn off format on save, then recommit

File: src/components/ValidatedInput/ValidCreatableSelect.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+/*
+ *
+ *  * Copyright (c) 2021 AccelByte Inc. All Rights Reserved.
```
Comment (rayhan-ab) [inline src/components/ValidatedInput/ValidCreatableSelect.tsx:3]: 2022?

File: src/components/styled-atlaskit/CreatableSelect/CreatableSelect.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2021 AccelByte Inc. All Rights Reserved.
```
Comment (rayhan-ab) [inline src/components/styled-atlaskit/CreatableSelect/CreatableSelect.tsx:2]: 2022?

File: src/components/index.ts [hunk 4]
```diff
@@ -9,6 +9,7 @@ export * from "./styled-atlaskit/DynamicTable/DynamicTable";
 export * from "./styled-atlaskit/Modal/Modal";
 export * from "./styled-atlaskit/Modal/utility";
 export * from "./styled-atlaskit/Select/Select";
+export * from "./styled-atlaskit/CreatableSelect/CreatableSelect";
```
Comment (rayhan-ab) [inline src/components/index.ts:12]: export ValidCreatableSelect as well?

File: src/components/ValidatedInput/ValidCreatableSelect.tsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  onChange,
+  errMessage,
+  optionalLabel,
+  isInvalid,
+  isDisabled = false,
+  isRequired = true,
+  isClearable,
+  tooltip,
+  selectDataQa,
+  isMulti = true,
```
Comment (rayhan-ab) [inline src/components/ValidatedInput/ValidCreatableSelect.tsx:29]: why isMulti is true by default while in CreatableSelect it's false?

File: CHANGELOG.md (hunk 1 above)
Comment (aris-fm) [reply CHANGELOG.md:7]: Yes, forgot to turn off prettier

File: src/components/ValidatedInput/ValidCreatableSelect.tsx (hunk 5 above)
Comment (aris-fm) [reply src/components/ValidatedInput/ValidCreatableSelect.tsx:29]: this is not intended, will update

File: src/components/index.ts (hunk 4 above)
Comment (aris-fm) [reply src/components/index.ts:12]: done

**Review by rayhan-ab — APPROVED** [review]

**Pull request merged**